lib/Cleantalk/ApbctWP/Firewall/SFW.php

```diff
@@ -293,6 +297,23 @@ public function diePage($result)
 
             $status     = $result['status'] === 'PASS_SFW__BY_WHITELIST' ? '1' : '0';
             $cookie_val = md5($result['ip'] . $this->api_key) . $status;
+    
+            /**
+             * Message about IP status
+             */
+            $message_ip_status = __(
+                'IP in the common blacklist',
+                'cleantalk-spam-protect'
+            );
+            $message_ip_status_color = 'red';
+            
+            if($result['status'] === 'PASS_SFW__BY_STATUS') {
+                $message_ip_status = __(
+                    'IP in the common whitelist',
+                    'cleantalk-spam-protect'
+                );
+                $message_ip_status_color = 'green';
+            }
 
```

Comment:
We shouldn't change the "non-sfw-test" output. Please, use "inherit" style for normal output.

We don't want to show MESSAGE_IP_STATUS for normal output either. Only for sfw-test.

Reply:
Fixed